contentScript.js

```diff
@@ -36,19 +36,11 @@ if (window.location.href.indexOf("bizjournals.com") !== -1) {
 	if (paywall) {
 		paywall.classList.remove('premium-content');
 		paywall.classList.add('full-content');
+		removeClassesByPrefix(paywall, 'QUnW');
 		var paras = paywall.querySelectorAll("p, span, h2, div");
```

Comment:
To follow ECMAScript standards you've to use `let` instead of `var`. You can read about differences [in this stackoverflow question](https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var)

```suggestion
		let paras = paywall.querySelectorAll("p, span, h2, div");
```